detection_rules/config.py

```diff
@@ -189,6 +189,7 @@ class RulesConfig:
     bbr_rules_dirs: Optional[List[Path]] = field(default_factory=list)
     bypass_version_lock: bool = False
     exception_dir: Optional[Path] = None
+    normalize_kql_keywords: bool = True
```

Comment:
Is this supposed to be `Optional`?

Reply:
It could be marked Optional, but was following the existing code for the `bypass_version_lock` which also is optional, but not marked so here. I think we will want them both to match. 

In effect, it is not Optional for the dataclass at the moment, but optional to have it set in the config. That being said, I think we should mark both as Optional, to be more clear for someone reading the code. 

Thoughts?

Reply:
I think based on our discussion we decided to leave this as not optional (it is optional in the config file but not in this config object). Happy to update change if desired.